src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseParticipationIntegrationTest.java

```diff
@@ -69,28 +74,81 @@ public void tearDown() {
         database.resetDatabase();
     }
 
-    @Test
+    private static Stream<Arguments> setArgumentsForGetParticipationWithLatestResult() {
```

Comment:
See reason given by Codacy.
```suggestion
    private static Stream<Arguments> argumentsForGetParticipationWithLatestResult() {
```